src/actions/User.js

```diff
@@ -0,0 +1,26 @@
+import keyMirror from 'utils/keyMirror'
+import origin from '../services/origin'
+
+export const UserConstants = keyMirror(
+  {
+    FETCH: null,
+    FETCH_SUCCESS: null,
+    FETCH_ERROR: null,
+  },
+  'USER'
+)
+
+export function fetchUser(address) {
+  return async function(dispatch) {
+    try {
+      const user = await origin.users.get(address)
+
+      dispatch({
+        type: UserConstants.FETCH_SUCCESS,
+        user: Object.assign({}, user, { address }),
```

Comment:
A shorthand for this is:

`user: { ...user, address }`

Reply:
I forgot about that one. 👍